### ISSUE TITLE
Bug in playlist view

### ISSUE THREAD

**Issue by michalsieron** [agersant/polaris-web#50]
So I tried saving some playlists and when I switched to one, that happened

![Screenshot_2020-05-27 Polaris](https://user-images.githubusercontent.com/54780569/83026727-a9267b80-a02f-11ea-945c-db2dd4dc9d71.png)

I checked and it has nothing to do with theme (after all it's just colors) and new playlist (same there).

**Comment by michalsieron** [comment]
Also because I am not sure whether to put it here or main polaris repo: when I save a playlist it just reloads page. It happens only first time after the page loads. After that it saves any playlist normally.

**Comment by agersant** [comment]
Thank you for the report! Did the issue in your screenshot happen a single time or can you reproduce it?
What OS and browser are you using?

**Comment by michalsieron** [comment]
I am running Polaris 0.12.1 and using Firefox 76.0.1 on Windows 10 on build 18363. I also checked it on Edge (both old and new) and it's also present there. Also Chrome on different PC seems to behave the same way. Unfortunately it is happening every single time I try to open a saved playlist.
Sidebar is unresponsive (or at least I cannot switch views from playlist).
I can press "Play" and "Delete" buttons in "browser", but cannot click any song title (although I can drag and drop them to put in queue).
In "right pane" I can click stuff normally (it's just shifted down). Here I tried to check some info about position of "right pane" (first one is before selecting playlist and second one is already shifted):
![boundingrect](https://user-images.githubusercontent.com/54780569/83130058-01b45200-a0de-11ea-986b-23bbeae088e1.png)
